plugins/data_migration/api/data_migration_helper.js

```diff
@@ -492,56 +499,56 @@ module.exports = function(my_db) {
-                    scripts.push('mongodump ' + dbstr + ' --collection feedback_widgets -q \'{ app_id: "' + appid + '"}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + ' --collection notes -q \'{ app_id: "' + appid + '"}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + '  --collection messages -q \'{ apps: "' + appid + '"}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + ' --collection cohortdata -q \'{ a: "' + appid + '"}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + ' --collection cohorts -q \'{ app_id: "' + appid + '"}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + ' --collection server_stats_data_points -q \'{ a: "' + appid + '"}\' --out ' + my_folder);
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'campaigndata', '-q', '{ a: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'campaigns', '-q', '{ app_id: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'crash_share', '-q', '{ app_id: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'feedback_widgets', '-q', '{ app_id: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'notes', '-q', '{ app_id: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'messages', '-q', '{ apps: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'cohortdata', '-q', '{ a: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'cohorts', '-q', '{ app_id: "' + appid + '"}', '--out', my_folder]});
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'server_stats_data_points', '-q', '{ a: "' + appid + '"}', '--out', my_folder]});
                     //concurrent_users
-                    scripts.push('mongodump ' + dbstr + ' --collection concurrent_users_max -q \'{$or:[{ app_id: "' + appid + '"},{ _id: {$in :["' + appid + '_overall", "' + appid + '_overall_new"]}}]}\' --out ' + my_folder);
-                    scripts.push('mongodump ' + dbstr + ' --collection concurrent_users_alerts -q \'{ app: "' + appid + '"}\' --out ' + my_folder);
+                    scripts.push({cmd: 'mongodump', args: [...dbargs, '--collection', 'concurrent_users_max', '-q', '{$or:[{ app_id: "' + appid + '"},{ _id: {$in :["' + appid + '_overall", "' + appid + '_overall_new"]}}]}', '--out', my_folder]});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4577914)